documentation/modules/rn-2.2.adoc

```diff
@@ -9,21 +9,33 @@ You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {vir
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
-//
-// .TBD
-//
-// TBD
-//
-// [id="new-features-and-enhancements-22_{context}"]
-// == New features and enhancements
-//
-// This release adds the following features and improvements.
-//
-// .TBD
-//
-// TBD
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+.Setting interval between precopies
+
+You can set the precopy interval.
+
+.Handling RHV VMs with USB enabled
```

Comment:
I don't think this is for USB enabled. That is a VM configuration setting. The description refers to mapped devices.

If you turn this into a known issue, the title will probably be:

`Migrating a {rhv-full} VM with a host USB device`